frontend/src/components/ProofingTab/GraphDiagram/Factories/CustomNodeFactory.tsx

```diff
@@ -25,9 +25,9 @@ export class CustomNodeFactory extends AbstractReactFactory {
             case NodeTypes.Via:
                 return <ViaNodeWidget engine={this.engine} model={event.model} />;
             case NodeTypes.Destination:
-                return <DestinationNodeWidget engine={this.engine} model={event.model} />;
+                return <DestinationNodeWidget engine={this.engine} model={event.model} forwardConnection={false}/>;
```

Comment:
issue (blocking): My understanding is that forwardConnection shouldn't always be false but it's rather on event.model.options.forward_connection.length

Reply:
Potentially we don't even need the prop as we already have access to the model which contains the forward_connection in the options but that's a detail. The issue I'm raising is the hardcoded false value